test/networkInterface.ts

```
@@ -293,47 +295,12 @@ describe('network interface', () => {
       );
     });
 
-    it('should return errors if the server responds with them', () => {
-      const swapi = createNetworkInterface('http://graphql-swapi.test/');
-
-      // this is a stub for the end user client api
-      const simpleRequest = {
-        query: `
-          query people {
-            allPeople(first: 1) {
-              people {
-                name
-              }
-          }
-        `,
-        variables: {},
-        debugName: 'People query',
-      };
-
-      return assert.eventually.deepEqual(
-        swapi.query(simpleRequest),
-        {
-          errors: [
-            {
-              message: 'Syntax Error GraphQL request (8:9) Expected Name, found EOF\n\n7:           }\n8:         \n           ^\n',
-              locations: [
-                {
-                  column: 9,
-                  line: 8,
-                },
-              ],
-            },
-          ],
-        }
-      );
-    });
-
```

Since we now use the query document for the Request, this test no longer makes any sense (i.e. "gql" should throw the same error before anything is sent to the server).